.ci/generate_bear_requirements.py

```diff
@@ -202,7 +214,7 @@ def sort_requirements(req_dict):
     if args.bear_dirs is not None:
         bear_dirs.extend(args.bear_dirs)
 
-    pip_reqs, npm_reqs, gem_reqs = (
+    pip_reqs, npm_reqs, gem_reqs, cabal_reqs = (
```

Comment:
As mentioned above, we should avoid returning tuples.
`get_all_requirements` could return something else...

alternatively...

```py
all_reqs = get_all_requirements(get_all_bears(bear_dirs))
pip_reqs, npm_reqs, gem_reqs = all_reqs[0:2]
....
requirements['cabal_requirements'] = get_gem_requirements(all_reqs[3])
```

Then it is easy to add another one afterwards.